### ISSUE TITLE
bug(Chips): Arrows navigation in NVDA browse mode automatically switched to focus mode

### ISSUE THREAD

**Issue by IConstructor** [angular/components#20761]
#### Reproduction
Steps to reproduce:
1. Go to https://material.angular.io/components/chips/examples
2. Open NVDA screen reader in browse mode
3. Start navigating in browse mode with a help of up/down arrows
4. Navigate with the help of arrows to chips select list
 
 
#### Expected Behavior

NVDA is reading information about the select list and keeps navigating in the browse mode

#### Actual Behavior

NVDA automatically switches to the focus mode and prevents user from further page exploring in the browse mode.

#### Environment

- Angular: 6+
- CDK/Material: Material components
- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): Windows

**Comment by IConstructor** [comment]
Also would like to add that nvda browse mode is reading only first selected chips element from the selected chips and then moving to the selection field (where it is switched to focus mode)

**Comment by crisbeto** [comment]
I could reproduce it, but I don't know of a way to tell the screen reader to go out of browse mode. The chip does seem to receive focus so I would've expected that to trigger focus mode automatically.

**Comment by IConstructor** [comment]
Hmm.. I just disabled "Automatic focus mode on focus changes" and the issue disappeared. So seems like this behavior is configurable.
But placeholder double pronunciation to me still look like issues.

**Comment by crisbeto** [comment]
The double placeholder issue sounds like something that we fixed for the plain input in https://github.com/angular/components/pull/10466, but it hasn't been applied to the chip input.

**Comment by zarend** [comment]
Hello,

I cannot produce this with legacy chips, and I also cannot produce with the mdc-based chips. @IConstructor does the issue persist for you?

Environment
- NVDA 2022.3
- Windows 10 Enterprise 21H2
- Chrome 107.0.5304.107 (Official Build) (64-bit)

-Zach

**Comment by zarend** [comment]
Hello @IConstructor ,

Thank you for reporting this issue. Unfortunately we are not able to take action if we cannot reproduce it. This might have already been fixed. If the issue persists, please provide reproduction steps or file a new issue report.

- Zach
